docker-compose-perf.yml

```diff
@@ -470,6 +472,36 @@ services:
       - agreement
       - all-services
       - transfers-test
+      - quotes-test
+
+  quoting-service:
+    image: "mojaloop/quoting-service:${QUOTING_SERVICE_VERSION}"
+    command: sh -c "/opt/app/config-modifier/run.js /opt/app/config/default.json /opt/app/config-modifier/configs/quoting-service-perf.js /opt/app/config/default.json && node src/index.js"
+    deploy:
+      mode: replicated
+      replicas: 1
+    networks:
+      - mojaloop-net
+    ports:
+      - "3002:3002"
```

Comment:
Can we disable exposing port just like other services?

Reply:
yep, forgot to comment it out.